CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g")
 SET(CMAKE_CXX_FLAGS_DEBUG  "-O0 -g")
 
 # flags required for SDL-3
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -fPIE -D_FORTIFY_SOURCE=2")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -march=native -fPIE -D_FORTIFY_SOURCE=2")
```

Comment:
I would move march to a separate set() statement as it is not required for SDL-3 as the comment suggests.